dockerfiles/examples/ml/build.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+##############################################################################
+#
+#    HPCC SYSTEMS software Copyright (C) 2020 HPCC Systems® .
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+##############################################################################
+
+# Use this script to build local images for Machine Learning HPCC Systems Docker images
+#
+
+LABEL=
+while getopts “hlt:” opt; do
+  case $opt in
+    l) TAGLATEST=1 ;;
+    t) LABEL=$OPTARG ;;
+    h) echo "Usage: incr.sh [options]"
+       echo "    -h                 Display help"
+       echo "    -l                 Tag the images as the latest"
+       echo "    -t                 Tag of base image hpccsystems/platform-core"
+       exit
+       ;;
+  esac
+done
+shift $(( $OPTIND-1 ))
+
+[[ -z ${LABEL} ]] && LABEL=latest
+
+
+ml_features=(
+  'scikit'
+  #'tf2'
+  #'tf2-gpu'
+)
+
+build_image()
+{
+  name=$1
+  ml_label=${name}
+  [[ "${LABEL}" != "latest" ]] && ml_label="${ml_label}-${LABEL}"
```

Comment:
confused by handling of "latest". Looks like it will do nothing?
Would clearer if the docker tag block and this block were all inside a != "latest" condition.

Reply:
Here is an example,  let say I want to build tf2. I can pass HPCC Docker Image tag as LEVEL or leave it as blank which will use latest HPCC Docker image.
If use a tag, for example community_7.10.8-1 I will build
1) hpccsystems/platform-core:tf2-community_7.10.8-1
2) if TAGLATEST=1  hpccsystems/platform-core:tf2

if use latest HPCC Docker image (LABEL=latest)
only  hpccsystems/platform-core:tf2 will be created.